committing.rst

```diff
@@ -127,12 +127,12 @@ is worth it, ask on python-dev.
 
 Third, ensure the patch is attributed correctly with the contributor's
 name in ``Misc/ACKS`` if they aren't already there (and didn't add themselves
-in their patch) and by mentioning "Patch by <x>" in the ``Misc/NEWS`` entry
+in their patch) and by mentioning "Patch by <x>" in the ``Misc/NEWS.d`` entry
 and the checkin message. If the patch has been heavily modified then "Initial
 patch by <x>" is an appropriate alternate wording.
 
 If you omit correct attribution in the initial checkin, then update ``ACKS``
-and ``NEWS`` in a subsequent checkin (don't worry about trying to fix the
+and ``NEWS.d`` in a subsequent checkin (don't worry about trying to fix the
```

Comment:
You can't point that out to the son of a former school teacher who almost had a minor in English and expect them not to fix it immediately. 😉